.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+# E203 is ignored for black
+# E402 is ignored because of te way we do relative imports
+extend-ignore = E203, E402
```

Comment:
It would be nice to have the descriptions of the flake rules here for people who do not memorize them by heart.

```suggestion
# Whitespace before ':' (E203) is ignored for black
# Module level import not at top of file (E402) is ignored because of the way we do relative imports
extend-ignore = E203, E402
```